File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioMapOfUserCondition.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdUserCondition;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioMapOfUserCondition extends StudioBase
{
  @Nullable
  public Boolean andOr;

  @Nullable
  public MetaIdUserCondition[] keys;

  @Nullable
  public Map<MetaIdUserCondition, StudioMapOfUserCondition> map;

  public MetaIdUserCondition metaId;

  @Nullable
  public StudioDtoUserConditionStatement statement;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfUserCondition.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdUserCondition
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoUserConditionStatement

interface StudioMapOfUserCondition : StudioBase
{
  val andOr: Boolean?
  val keys: List<MetaIdUserCondition>?
  val map: Map<MetaIdUserCondition, StudioMapOfUserCondition>?
  val metaId: MetaIdUserCondition
  val statement: StudioDtoUserConditionStatement?
}